plugin/symbols.py

```diff
@@ -75,12 +75,15 @@ def __init__(self, view: sublime.View) -> None:
         self.is_first_selection = False
 
     def run(self, edit: sublime.Edit) -> None:
+        self.view.settings().set('lsp_supress_input', True)
         session = self.best_session(self.capability)
         if session:
             session.send_request(
-                Request.documentSymbols({"textDocument": text_document_identifier(self.view)}), self.handle_response)
+                Request.documentSymbols({"textDocument": text_document_identifier(self.view)}),
+                self.handle_response, self.handle_response_error)
 
     def handle_response(self, response: Any) -> None:
+        self.view.settings().erase('lsp_supress_input')
```

Comment:
Hmm... `handle_response` runs in some thread somewhere. That means that theoretically what could happen is: the view setting is erased, then the user types a character, and only then the quickpanel is shown.

In order to guarantee atomicity here, we should run the handler on the UI thread with `sublime.set_timeout`.